Bottom: aea155e9d5639916fd5f400be1d9b7576e54038a
Top:    5601d9310c615672d8df729a0f8a8373470edc56
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-05-08 20:15:24 +0100

Refresh of add-action-to-set-netclass

---

diff --git a/tools/gnet-pcbfwd.scm b/tools/gnet-pcbfwd.scm
index e225704..9067978 100644
--- a/tools/gnet-pcbfwd.scm
+++ b/tools/gnet-pcbfwd.scm
@@ -57,8 +57,8 @@
 (define (pcbfwd:each-net netnames port)
   (if (not (null? netnames))
       (let ((netname (car netnames)))
-	(pcbfwd:net-class netname port)
 	(pcbfwd:each-pin netname (gnetlist:get-all-connections netname) port)
+	(pcbfwd:net-class netname port)
 	(pcbfwd:each-net (cdr netnames) port))))
 
 (define (pcbfwd:each-attr refdes attrs port)
